#GPS
float64 latitude
float64 longitude

float64 track
float64 speed
float64 climb

# Point position
float64 pos_x
float64 pos_y
float64 pos_z

# Quaternion orientation
float64 orient_x
float64 orient_y
float64 orient_z
float64 orient_w

# linear twist
float64 linear_x
float64 linear_y
float64 linear_z 

# angular twist
float64 angular_x
float64 angular_y
float64 angular_z 



int16 label
